bvm/ballerina-runtime/src/main/resources/META-INF/native-image/ballerina/runtime/native-image.properties

```diff
@@ -0,0 +1,2 @@
+
+-H:ResourceConfigurationResources=${.}/resource-config.json
```

Comment:
The format should be :
```
Args = -H:ResourceConfigurationResources=${.}/resource-config.json
```

Anyway we don't need this argument/ this file. If we name the resource configuration file as `resource-config.json` it will be automatically pickup by the native image tool.